src/formatDate.js

```diff
@@ -50,7 +50,66 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat[3];
+  const upgradeSeparator = toFormat[3];
+  const dateArray = date.split(separator);
+  let yearYYYY;
```

Comment:
Not the best variable name.
I would suggest `yearLong` and `yearShort` instead.